### ISSUE TITLE
Setup Netlify for Docusaurus deployment

### ISSUE THREAD

**Issue by pavithraes** [conda-incubator/conda-store#407]
Additionally, it'll be nice to have a docs preview on pull requests for quicker code-review.

Ref: https://docs.readthedocs.io/en/stable/pull-requests.html

**Comment by trallard** [comment]
Waiting on #534 (discussion)

**Comment by pavithraes** [comment]
Let's work on https://github.com/conda-incubator/conda-store/pull/534#issuecomment-1676388009 in a new PR against this issue. :)

**Comment by costrouc** [comment]
FYI I would guess soon github will support this feature as well https://github.com/actions/deploy-pages/blob/368bf1aa22b9265440e251b9216520e3f91c2d5b/action.yml#L36

**Comment by trallard** [comment]
Done now and deployment is working as expected. 
note that until we migrate the docs this will remain in a separate URL
https://64f2225a9fd51e0a070cc500--kaleidoscopic-dango-0cf31d.netlify.app/
